Lab2.ipynb

```diff
@@ -0,0 +1,830 @@
+      "text/plain": [
+       "<Figure size 720x504 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.rcParams[\"figure.figsize\"] = (10,7)\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "x = range(0,50)\n",
+    "mu = 3\n",
+    "prob = stats.poisson.pmf(x, mu) \n",
+    "ax.plot(x,prob, label='poisson pdf')\n",
```

Comment:
I think it will be nice if you could add titles for your plots using `plt.title("")` to explain what the plots are plotting. For example, original Poisson distribution or summed Poisson distribution for X days.